rs/types/management_canister_types/src/lib.rs

```diff
@@ -2251,6 +2251,131 @@ impl FromStr for SchnorrKeyId {
     }
 }
 
+/// Types of curves that can be used for threshold key derivation (vetKD).
+/// ```text
+/// (variant { bls12_381_g2; })
+/// ```
+#[derive(
+    Copy,
+    Clone,
+    Eq,
+    PartialEq,
+    Ord,
+    PartialOrd,
+    Hash,
+    Debug,
+    CandidType,
+    Deserialize,
+    EnumIter,
+    Serialize,
+)]
+pub enum VetKdCurve {
+    #[serde(rename = "bls12_381_g2")]
+    #[allow(non_camel_case_types)]
+    Bls12_381_G2,
+}
```

Comment:
I think we normally format the derives on one line?

Reply:
@adambratschikaye, I'm resolving this conversation based on the above comment. Let me know if there are further questions.